UniqueInOrder.js

```diff
@@ -7,17 +7,17 @@ uniqueInOrder('AAAABBBCCDAABBB') == ['A', 'B', 'C', 'D', 'A', 'B']
 uniqueInOrder('ABBCcAD')         == ['A', 'B', 'C', 'c', 'A', 'D']
 uniqueInOrder([1,2,2,3,3])       == [1,2,3]
 */
-var uniqueInOrder=function(iterable){
-    //your code here - remember iterable can be a string or an array
-    if(iterable.length ===0){
-      return []
+var uniqueInOrder = (iterable) => {
```

Comment:
this should be a const aince its never changing. for reference https://www.developer.com/languages/javascript/javascript-var-let-const-variable-declaration/#:~:text=var%2C%20let%2C%20and%20const%20Variable%20Declaration%20in%20JavaScript&text=Hoisting%20means%20that%20the%20variable,to%20the%20immediate%20enclosing%20block.